src/Views/batchprocessor.html

```diff
@@ -163,6 +173,15 @@ <h3>Select Basin Characteristics</h3>
 
         <!-- Start batch status -->
         <div id="batchStatus" ng-if="vm.selectedBatchProcessorTabName==='batchStatus'">
+            <!-- Start instructions -->
+            <div id="batchStatusInstructions"> To check batch status: <br>
+                <ul style="list-style-type: circle;">
+                    <li>Enter your email address in the Email Address box</li>
+                    <li>When the results are ready for download, a hyperlink will appear in the <i>Results</i> field</li>
+                    <li>A batch can be deleted by clicking on the trash can icon and answering Yes to the question that pops up</li>
+                </ul>
+            </div>

```

Comment:
Some suggestions for clarity / consistency with other tab.
```suggestion
            <div> 
                Batches may take several hours or days to complete. Use this tab to check the status and results of your batches.
                <br>
                <br>
                <b>How to Check Batch Status</b>
                <ol>
                    <li>Enter your email address.</li>
                    <li>Batch status will appear in the <i>Status</i> column.</li>
                    <li>When results are ready for download, a hyperlink will appear in the <i>Results</i> column.</li>
                    <li>If you would like to remove a batch from the queue, click on the trash can icon.</li>
                </ul>
            </div>

```

Reply:
I incorporated both of your suggestions and the ordered lists look much better!